src/Handler/StreamHandler.php

```diff
@@ -284,7 +284,7 @@ function () use ($context, $params) {
 
         return $this->createResource(
             function () use ($request, &$http_response_header, $context) {
-                $resource = fopen($request->getUri(), 'r', null, $context);
+                $resource = fopen($request->getUri(), 'r', '', $context);
```

Comment:
yes I used bool at the beginning, works the same, but an empty string preserving the type on the surface kind of felt more approriate